Registry/Registry.EM_COMMON

```diff
@@ -339,9 +339,9 @@ i1       real   tke_tend       ikj     dyn_em      1         -
 
 # Pressure and Density
 state    real   p              ikj     dyn_em      1         -      irh       "p"           "perturbation pressure"         "Pa"
-state    real   al             ikj     dyn_em      1         -      r         "al"          "inverse perturbation density"  "m3 kg-1"
+state    real   al             ikj     dyn_em      1         -      ir         "al"          "inverse perturbation density"  "m3 kg-1"
 state    real   alt            ikj     dyn_em      1         -      r         "alt"         "inverse density"               "m3 kg-1"
-state    real   alb            ikj     dyn_em      1         -      rdus      "alb"         "inverse base density"          "m3 kg-1"
+state    real   alb            ikj     dyn_em      1         -      irdus      "alb"         "inverse base density"          "m3 kg-1"
```

Comment:
@smileMchen 
Ming,
Keep the existing column alignments

Reply:
@davegill 
Dave,
It is lined up. Thanks.